projects/storefrontapp-e2e-cypress/cypress/integration/regression/customer-ticketing/ticket-listing.e2e-spec.ts

```diff
@@ -47,6 +48,24 @@ describe('ticket listing', () => {
         customerTicketing.verifyStatusOfTicketInList();
       });
 
+      it('should still show the ticket in the list when status is In Process', () => {
+        const testTicketDetails: TestTicketDetails = {
+          subject: 'changing status',
+          message: 'status will change',
+          category: TestCategory.complaint,
+        };
+
+        customerTicketing.loginRegisteredUser();
+        customerTicketing.visitElectronicTicketListingPage();
+        customerTicketing.createTicket(testTicketDetails);
+        customerTicketing.clickTicketInRow();
```

Comment:
Could you please add a parameter to know which row you are clicking without the need to navigate to the function's implementation